### PR TITLE
readme suggestions

### PR DESCRIPTION
**Pull request by sndrs**
readme suggestions. inline was getting really noisy so apologies for PR. 

i've tried to keep it as terse and generic as possible, see what you reckon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,118 @@
 # Consent Management Platform
 
-Welcome to the Consent Management Platform, a library to manage privacy frameworks across \*.theguardian.com, from messaging to reporting the privacy state. All exports include Typescript definitions.
+> Consent management for `*.theguardian.com`.
 
-## What utilities does this offer?
+The CMP applies the CCPA to users in the USA, and TCFv2 to everyone else.
 
-### TCF and CCPA privacy frameworks
+![Types](https://img.shields.io/npm/types/@guardian/consent-management-platform)
```
Comment (ripecosta) [inline README.md:7]: This link takes me to an image. Is that right?

Comment (sndrs) [reply README.md:7]: github just does that for any images you add I think?

Comment (ripecosta) [reply README.md:7]: I thought you wanted to link it to a different page showing the type definition. Kind of like you link to our chat room below. Is that not the case?

Comment (sndrs) [reply README.md:7]: it was mainly just show the badge. is there a place it could link to?

Comment (ripecosta) [reply README.md:7]: Oh I see! All makes sense now. To my knowledge there isn't a place we can link to to show the type definitions. But I've been known to be wrong before 😜so I thought you knew something I didn't.

**Pull request merged**